src/components/personside/Personside.tsx

```diff
@@ -24,10 +26,17 @@ class Personside extends React.PureComponent<PersonsideProps> {
     }
 
     render() {
+
+        const oversikt = (
+            <VisittkortContainer/>
+        );
+
+        const dialogpanel = (
+            <ComponentPlaceholder name={'Dialog Panel'} />
+        );
+
         return (
-            <div>
-                <VisittkortContainer/>
-            </div>
+            <MainLayout oversikt={oversikt} dialogpanel={dialogpanel}/>
```

Comment:
Er `MainLayout` komplisert nok til at vi trenger denne typen parameterisering?

Reply:
Var litt usikker. Grunnen til at jeg gjorde dette var for å sørge for at riktige komponenter styles på riktig måte. Et alternativ er å sende inn komponentene som children i en gitt rekkefølge som sørger for at de får riktig styling.